src/type-helper.d.ts

```diff
@@ -180,3 +180,22 @@ enum W3CGroupType {
   "ig",
   "wg",
 }
+
+type Person = {
+  name?: string;
+  w3cid?: string|number;
+  mailto?: string;
+  url?: string;
+  orcid?: string;
+  company?: string;
+  companyURL?: string;
+  note?: string;
+  retiredDate?: string;
+  extras?: PersonExtras;
```

Comment:
I was hoping TypeScript to catch this 🤔 
```suggestion
  extras?: PersonExtras[];
```